cwms_radar_api/src/main/java/cwms/radar/data/dto/CwmsDTOPaginated.java

```diff
@@ -0,0 +1,128 @@
+package cwms.radar.data.dto;
+
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.Base64.Decoder;
+import java.util.Base64.Encoder;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import javax.xml.bind.annotation.*;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+/***
+ * Provides a framework for a paginated result set.
+ * Implementation details are subject to inheritors.
+ * @author Daniel Osborne
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlAccessorOrder(XmlAccessOrder.ALPHABETICAL)
+public abstract class CwmsDTOPaginated implements CwmsDTO {
```

Comment:
Making this one class all paginated things derived from was a great idea. Thanks.